GuideForNewDevelopers.md

```diff
@@ -0,0 +1,579 @@
+
+```sh
+cd physionet-build
+```
+Now create the virtual environment with the following command in the same terminal
+```sh
+python3 -m venv env
+```
+
+##### 2.1.7.2 Activate the Virtual Environment
+
+In the same terminal from  step [2.1.7.1](#2171-create-a-virtual-environment), enter the following command to activate the virtual environment
+
+```sh
+source env/bin/activate
+```
+
+
+#### 2.1.8 Install requirements
+
+Now that we have the Virtual Environment setup, let's install the python libraries needed for the project.
+
+In the same terminal from step [2.1.7.2](#2172-activate-the-virtual-environment), enter the following command to install the requirements
+
+```sh
+pip install -r requirements.txt
+```
+
+#### 2.1.9 Run the project
+We now have everything set up to run the project locally. Let's go ahead and set up the project
+
+##### 2.1.9.1 Open a terminal and navigate inside the project directory
+
+```sh
+cd physionet-build
+```
+
+
+
+##### 2.1.9.2 Activate virtual python environment.
+
+```sh
+source env/bin/activate
+```
+
+##### 2.1.9.3 Copy .env.example file to .env
+
+```sh
+cp .env.example .env
+```
+
+
+
+##### 2.1.9.4 Run the following commands to setup the database
+
+  - Run: `python3 manage.py resetdb` to reset the database with the latest applied migrations.
+
+  - Run: `python3 manage.py loaddemo` to load the demo fixtures set up example files.
+  
+  - Run: `python3 manage.py compilestatic` to compile the css files
+
+#### 2.1.10 Start the server(Finally)
+
+Enter the following command to start the server
+
+```sh
+python3 manage.py runserver
+```
+
+The local development server will be available at [http://localhost:8000](http://localhost:8000).
+
+
+
+
+### 2.2 Windows
+
+
+#### 2.2.1 Install Python3
+
+*Note: Check if you already have a python3 installed on your system(if you already have a latest version of python3, you wont to do do the step 1.2.1), to check enter the following on CMD*
+
+```cmd
+python --version
+```
+
+If you don't have the latest version of python, Please download and install the latest python from the [Official Python Website](https://www.python.org/downloads/). 
+
+1. Navigate to the Downloads tab for Windows.
+2. Choose the latest Python 3 release
+3. Choose the Windows x86 executable installer if you are using a 32-bit installer or if you have a 64-bit system, then download Windows x86-64 executable installer. 
+4. Run the executable and install python with the default options
+   1. Don't forget to select the `Add Python x.x to PATH` option
+   
+   Here is a detailed [guide](https://realpython.com/installing-python/#how-to-install-python-on-windows) if you need further help
+
+Open cmd and enter the following commands to verify you have installed python3
+
+
+```cmd
+python --version
+```
+
+#### 2.2.2 Install git Bash
+Download git bash from the official website and install it with the default settings
```

Comment:
[official website](https://git-scm.com/download/win)

instead of having the "raw" link